src/Components/Quotecard/Quotecard.tsx

```diff
@@ -10,6 +10,7 @@ export function Quotecard(props: QuotecardProps) {
     return (
         <div className="quotecard">
             <p className="quoteText">{props.quote}</p>
+            <br/>
```

Comment:
We should probably fix this in the CSS - that way we keep our HTML nice and simple.
`p` tags are normally block elements so would have their own lines by default.

I'm guessing they are in a flex now though? Making the width `100%` will probably fix this.